src/main/java/org/takes/rq/RqForm.java

```diff
@@ -64,6 +64,7 @@
  * @see <a href="http://www.w3.org/TR/html401/interact/forms.html">
  *     Forms in HTML</a>
  * @see org.takes.rq.RqGreedy
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@dalifreire please create a puzzle to refactor this class and get rid the supression